File: 724-find-pivot-index/find-pivot-index.py
class Solution:
    def pivotIndex(self, nums: List[int]) -> int:
        total=sum(nums)
        summ=0

        for i in range(len(nums)):
            if summ==total-summ-nums[i]:
                return i
            
            summ+=nums[i]

        return -1

File: 724-find-pivot-index/find-pivot-index.cpp
class Solution {
public:
    int pivotIndex(vector<int>& nums) {
        int summ=0,total=0;

        for(auto num: nums) total+=num;

        for(int i=0; i<nums.size(); i++){
            if (summ==total-summ-nums[i])
                return i;
            
            summ+=nums[i];
        }

        return -1;
    }
};